OrPinhas_and_AvivShabtay_Project/OrPinhas_and_AvivShabtay/OrPinhas_and_AvivShabtay/Hotel.h

```diff
@@ -35,11 +35,11 @@ class Hotel
 		delete[] hotelName;
 	};
 
-//	friend ostream& operator<<(ostream& os, Hotel& hotel) const
-//	{
-//		os << 'List Of Crew For Current Shift:' << employee.crewList;
-//		return os;
-//	};
+	//	friend ostream& operator<<(ostream& os, Hotel& hotel) const
```

Comment:
This is fine good usage of frient.
The `ostream` object is outer object to this class and here in this case, using `friend` simplify the access to the class members and let you print the object data easily.